docs/content/integrations/duckdb/using-duckdb-with-dagster.mdx

```diff
@@ -30,45 +29,203 @@ To complete this tutorial, you'll need:
 
 ---
 
-## Step 1: Configure the DuckDB I/O manager
+This tutorial is divided in two parts. Both parts will create the same assets, but how the data is stored in DuckDb will differ. In [Part 1](#part-1-using-the-duckdb-resource) you will use the DuckDB resource to manually execute SQL queries to create tables and interact with those tables. In [Part 2](#part-2-using-the-duckdb-io-manager) you will transfer the responsibility for generating the SQL to create and interact with tables to the DuckDB I/O manager. When writing your own assets, you may choose one or the other (or both) approaches depending on your storage requirements. See [When to use I/O managers](TODO-LINK) to learn more.
```

Comment:
A few suggestions for this paragraph:

1. It should be at the end of the intro, just before the **Prerequisites**. It's in a weird no-man's land right now
2. Consider breaking this up into a list to make it easier to read - ex:

> This tutorial is divided into two parts:
> - In Part 1...
> - In Part 2...
>
> When writing your own assets, you may choose one or the other (or both) approaches...